### PR TITLE
Feature/azure updates

### PR DESCRIPTION
**Pull request by medgar-nitric**

### CODE REVIEW

**Review by davemooreuws — DISMISSED** [review]
LGTM

File: docs/intro/cloud-providers/azure.mdx [hunk 1]
```diff
@@ -46,12 +46,21 @@ az login
 
 When you're ready to deploy your application to Azure, you can use the Azure CLI Plugin to automate the deployment process.
 
-Use the `deploy` command from the root of your project to start an interactive deployment process, with prompts for Azure region, org. name and admin email:
+Use the `deploy:azure` command from the root of your project to start an interactive deployment process, with prompts for Azure region, org. name and admin email:
```
Comment (jyecusch) [inline docs/intro/cloud-providers/azure.mdx:49]: We should probably provide information about what the Org name and admin email are used for. They're unique to Azure, so a link to the docs that explain what these values are for would be useful here.

**Review by jyecusch — CHANGES_REQUESTED** [review]
One note for Azure, otherwise looks good.

File: docs/intro/cloud-providers/azure.mdx (hunk 1 above)
Comment (medgar-nitric) [reply docs/intro/cloud-providers/azure.mdx:49]: done

**Review by jyecusch — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**